### ISSUE TITLE
Bundling of external audio assets [QUESTION]

### ISSUE THREAD

**Issue by andrerfneves** [Microsoft/react-native-code-push#962]
### Description

This is a question for clarification before setting up CodePush within my Application. I want to use something like `react-native-sound` to play audio files in my app. Since I'll be `import`ing them into my JS files, I believe these assets will be coupled within my application bundle (just like how images are handled). Am I correct to assume such a premise? If I were to update/change/delete these audio files from my application and then `code-push` it, would those assets be bundled too? I see the `assets-folder` property in the CLI, but want to make sure I'm correct before settling with CodePush for OTA updates.

Thanks in advance!

### Reproduction
N/A

### Additional Information
N/A

**Comment by max-mironov** [comment]
Hello @andrerfneves, that's a good question. So it looks like that approach should work.
I've tried a quick demo on iOS with `react-native-sound` and it works great for me (meaning that I was able to update mp3 files with code push release).

I've used similar approach as with Images and call `require` for loading asset (also created  "audio" folder in the root of the app).

Here is my constructor init code:
`var mySound = new Sound(require(‘./audio/a3734.mp3’), ‘’, (error) => {`

So to answer your question - yes, assets will be coupled within your application bundle if you reference them via `require`.
